validation/validation.go

```diff
@@ -112,7 +112,7 @@ type Validation struct {
 	RequiredFirst bool
 
 	Errors    []*Error
-	ErrorsMap map[string]*Error
+	ErrorsMap map[string][]*Error
```

Comment:
Why change to error slice? you want to support multi error for one field?

Reply:
Yes. If field has many errors, we can show it all.